web/base_viewer.js

```diff
@@ -1475,6 +1491,9 @@ class BaseViewer {
     }
     this._setCurrentPageNumber(pageNumber, /* resetCurrentPageView = */ true);
     this.update();
+    if (this._currentScaleValue && isNaN(this._currentScaleValue)) {
```

Comment:
If I look at https://github.com/mozilla/pdf.js/blob/d784af3f38d5604b653c06fb40a6fe6a021d45d3/web/base_viewer.js#L1412-L1416 this happens _before_ `_setCurrentPageNumber` and `update`. Shouldn't that be done here as well?

Reply:
> Shouldn't that be done here as well?

That's correct, since otherwise there's a risk that the active page changes.